pkg/sfu/downtrack.go

```diff
@@ -583,14 +603,37 @@ func (d *DownTrack) CreateSenderReport() *rtcp.SenderReport {
 	}
 }
 
-func (d *DownTrack) UpdateStats(packetLen uint32) {
-	d.octetCount.add(packetLen)
-	d.packetCount.add(1)
+func (d *DownTrack) UpdatePrimaryStats(packetLen uint32) {
+	primaryStats, _ := d.primaryStats.Load().(*PacketStats)
+
+	primaryStats.octets += packetLen
+	primaryStats.packets += 1
+
+	d.primaryStats.Store(primaryStats)
+}
+
+func (d *DownTrack) UpdateRtxStats(packetLen uint32) {
+	rtxStats, _ := d.rtxStats.Load().(*PacketStats)
```

Comment:
if need a accuracy stat here, will not get correct values when called concurrency, need use atomic.int in each single field

Reply:
Yeah, can be tightened even further, but this is not that important to be super accurate. So, left it simpler for now.